ProcessLib/DensityDrivenFlow/CreateDensityDrivenFlowProcess.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PROCESS_LIB_CREATE_DensityDrivenFlowPROCESS_H_
```

Comment:
If an appropriate process name is choosen I'll will update the guards using upper case characters.